README.md

```diff
@@ -23,6 +23,29 @@ function swap(arr1, arr2) {
   // -- YOUR ANSWER HERE --
 }
 ```
+```
+function swap (first_arr,sec){
+  var temp = []
+  
+  for (var x = 0; x < first_arr.length; x++){
+    
+    temp[x] = first_arr[x]
+  }
+  
+  
+  for (var i = 0; i<first_arr.length; i++){
+  first_arr[i] = sec[i]
+  
+ }
+ 
+ for (var y = 0; y < sec.length; y++){
+    
+    sec[y] = temp[y]
+  }
+ 
+
+}
```

Comment:
95% of the way there.  Good job!
What about if the arrays have different lengths?